### PR TITLE
[GH-15875] Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 564

### PR DESCRIPTION
**Pull request by xaylax**
#### Summary
Removes the line: 
image

from utils/utils.jsx and replaces it with a new background variable under the app__body section in sass/layout/_navigation.scss

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/15875

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
LGTM :+1:

File: sass/layout/_navigation.scss [hunk 1]
```diff
@@ -201,4 +201,8 @@
     }
 }
 
-
+.app__body {
+    .navbar-right__icon {
+        background: rgba(var(--sidebar-header-text-color-rgb), 0.2);
```
Comment (ali-farooq0) [inline sass/layout/_navigation.scss:206]: let's move this up under an existing selector on https://github.com/mattermost/mattermost-webapp/pull/6745/files#diff-33df6171c54ae8787ddb1c33f38d4496R91.

**Review by ali-farooq0 — CHANGES_REQUESTED** [review]
Thanks for your contribution @xaylax! 🎉 Recommend moving the css into an existing selector.

File: sass/layout/_navigation.scss (hunk 1 above)
Comment (xaylax) [reply sass/layout/_navigation.scss:206]: @ali-farooq0 thanks for the comment! Would you be able to clarify what you mean by an existing selector? Do you mean putting it under here?
https://github.com/mattermost/mattermost-webapp/blob/ff6dcbaa312c1aa8f5de57b0672fc8f8dd083f86/sass/layout/_navigation.scss#L90

Comment (ali-farooq0) [reply sass/layout/_navigation.scss:206]: Yes. To avoid creating this new section

Comment (xaylax) [reply sass/layout/_navigation.scss]: Ok, done!

**Review by ali-farooq0 — APPROVED** [review]
Nice work @xaylax! 🎉

**Comment by hanzei** [comment]
Thanks for the contribution @xaylax :tada:

**Pull request merged**